<launch>
  <param name="function" value="3" />
  <param name="creg_min_cluster_size" value="400" />
  <param name="creg_max_cluster_size" value="500" />
  <param name="distance_thresh" value="0.01" />
  <param name="point_color_thresh" value="50" />
  <param name="region_color_thresh" value="5" />
  <node pkg="perception" type="point_cloud_demo" name="point_cloud_demo" output="screen">
    <remap from="cloud_in" to="mock_point_cloud" />
  </node>
</launch>
